### DISCUSSION TITLE
function calling pipelines

### DISCUSSION THREAD

**Discussion by jdevsan** [open-webui/pipelines#213] [General]
hey can anyone expalin how to set up function calling pipelines? i got my own model with langchain for tools using model phi3-medium-128k, I cannot figure out how to adapt this to pipelines..

**Comment by zabirauf** [comment]
I recently had a writeup on how to setup Function Calling pipeline (aka Tools in pipeline vernacular). 
See the Tools pipeline in the blog post. 
https://zohaib.me/extending-openwebui-using-pipelines/
